diagnostic_updater/include/diagnostic_updater/diagnostic_updater.hpp

```diff
@@ -369,35 +361,44 @@ class Updater : public DiagnosticTaskVector
   /**
```

Comment:
Same as with python: if force_udpate is kicked out, documentation block needs to be updated.